manager/integration/tests/common.py

```diff
@@ -1061,6 +1062,19 @@ def write_device_random_data(dev, position={}):
     }
 
 
+def write_volume_spec_data(volume, spec_data):
```

Comment:
The old `write_volume_data` is wrong, it's more like `write_pod_volume_data`.

This one should be `write_volume_data`. Let's correct the naming.